src/lib.rs

```diff
@@ -24,6 +24,7 @@
 #![deny(unstable_features)]
 #![deny(bare_trait_objects)]
 #![allow(clippy::manual_non_exhaustive)] // MSRV is 1.36
+#![allow(clippy::derivable_impls)]
```

Comment:
```suggestion
```

Reply:
Removing this fails the CI run on Windows. Should I rather keep this clippy flag or remove the affected implementations of `Default`?

Reply:
Ok, I can handle this later.